Minio/DataModel/Args.cs

```diff
@@ -36,6 +38,7 @@ public static Hashtable CloneHashTable(Hashtable h)
         }
         public Args()
         {
+            this.RequestMethod = Method.HEAD;
```

Comment:
is this required?

Reply:
If it is not assigned, the request is HEAD.